src/client/package.json

```diff
@@ -22,7 +22,8 @@
   "scripts": {
     "clean": "rm -rf dist",
     "start": "npm run dev",
-    "lint": "eslint src",
+    "dev:local": "npm run dev -- --endpoint local",
```

Comment:
Shouldn't this be config we pass?  
Just as a demo tool this is pretty esoteric?